tests/test_ssl.py

```diff
@@ -2522,6 +2522,63 @@ def test_get_peer_cert_chain_none(self):
         interact_in_memory(client, server)
         assert None is server.get_peer_cert_chain()
 
+    def test_get_verified_chain(self):
+        """
+        `Connection.get_verified_chain` returns a list of certificates
+        which the connected server returned for the certification verification.
+        """
+        chain = _create_certificate_chain()
+        [(cakey, cacert), (ikey, icert), (skey, scert)] = chain
+
+        serverContext = Context(TLSv1_METHOD)
```

Comment:
Could you change all these to `SSLv23_METHOD`? `TLSv1_METHOD` won't work in Ubuntu 20.04 (which is what our CI just switched to) because TLS 1.0 is disabled. `SSLv23_METHOD` is an awful name but will allow up to TLS 1.3.